api/webpack.config.js

```diff
@@ -18,8 +17,8 @@ module.exports = {
     ],
   },
   output: {
-    path: path.join(__dirname, 'static/ombpdf/js/'),
-    publicPath: 'static/ombpdf/js/',
+    path: path.join(__dirname, 'webpack-static/bundles/js/'),
+    publicPath: 'static/bundles/js/',
```

Comment:
Just to make sure I understand this, it shouldn't be `webpack-static` because Django will serve all the static files under the `/static` path?

Reply:
oh, I just called it `webpack-static` so it makes sense to devs what's in the folder, when they're looking at the directory tree on their local systems... does that answer your question? I might be misinterpreting your question.